examples/native-hybrid-app/android/app/src/main/java/com/datadoghq/hybrid_flutter_example/HybridApplication.kt

```diff
@@ -89,9 +108,15 @@ class HybridApplication : Application() {
                 componentPredicate = FlutterExcludingComponentPredicate()
             ))
         Rum.enable(rumConfiguration.build())
-        val traceConfig = TraceConfiguration.Builder()
-        Trace.enable(traceConfig.build())
 
+        // ... and NDK crash reporting (this is optional. See documentation for more details).
+        // https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/android/error_tracking
+        // NdkCrashReporting.enable()
+
+
+        // Once Datadog is fully initialized, you can run flutterEngine.run(),
+        // which will call Flutter's `main` method, which will look for an
+        // existing Datadog instance to attach to.
```

Comment:
ultra nit to avoid repetition of "which"
```suggestion
        // Once Datadog is fully initialized, you can run `flutterEngine.run()`.
        // This calls Flutter's `main` method, which will look for an existing Datadog instance to attach to.
```